# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/tsa/ctwm/manpage.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./ctwm.man.orig	2001-12-11 16:38:52.000000000 +0100
+++ ./ctwm.man	2003-04-26 20:06:36.000000000 +0200
@@ -1,13 +1,10 @@
 .de EX		\"Begin example
-.ne 5
 .if n .sp 1
 .if t .sp .5
 .nf
-.in +.5i
 ..
 .de EE
 .fi
-.in -.5i
 .if n .sp 1
 .if t .sp .5
 ..
